tools/devtool

```diff
@@ -575,11 +674,24 @@ cmd_strip() {
 # Please see `$0 help` for more information.
 #
 cmd_test() {
+    # Inherited from `shield.py`.
+    ESKIP=12
+    ESINGLENUMANODE=13
+    trap cleanup_shield EXIT
 
     # Parse any command line args.
     while [ $# -gt 0 ]; do
         case "$1" in
             "-h"|"--help")      { cmd_help; exit 1; } ;;
+            "-s"|"--shield")
+                shift
+                validate_shield $1
+
+                # Pattern match on the shield: {TESTS_NUMA_NODE_ID};{CPUS_COUNT}.
+                [[ "$1" =~ ^(0|[1-9][0-9]*)\;([1-9][0-9]*)$ ]] && \
+                TESTS_NUMA_NODE_ID=${BASH_REMATCH[1]} && \
+                CPUS_COUNT=${BASH_REMATCH[2]}
+                ;;
```

Comment:
I am not sure having custom flags or parsing flags is a better approach in this case. It will provide clearer arguments but it will be unusual, in my opinion. The plan here is to provide a digestible pattern but also to provide ways to extend it for scenarios where we must set shields cross NUMA nodes. Working with the provided pattern, we can easily extend it for shields cross numa nodes like so: `{TESTS_NUMA_NODE_ID1,TESTS_NUMA_NODE_ID2,...};{CPUS_COUNT1,CPUS_COUNT2,....}`. This is somehow similar to the jailer `--cgroups` param pattern added in #2012. At the moment, I am not sure where the cross numa node scenario might be useful, but it might be in the near future. The support can be added later on a need basis.

Reply:
Cross numa-node cpusets could be created by specifying multiple numa nodes: 

```
--node_0 1 --node_1 1 
```

Also, now that I think about it, we should also specify where the memory should come from. We might have scenarios where we want to allocate CPUs from node 0 and memory from node 1. Maybe not in the performance tests, but we might want to run some manual tests with such configurations sometimes. I definitely had to run such tests before.